gtda/images/filtrations.py

```diff
@@ -184,7 +184,7 @@ class RadialFiltration(BaseEstimator, TransformerMixin):
         dimension of the images of the collection.
 
     radius : float or None, default: ``None``
-        The radius of the ball centered in `center` inside which
+        The radius of the ball centered in :param:`center` inside which
```

Comment:
There are many instances of :param: still in `filtrations.py`, could you remove them?